src/librustdoc/json/conversions.rs

```diff
@@ -34,23 +33,26 @@ impl JsonRenderer<'_> {
                 did.map(|did| (link.clone(), from_def_id(did)))
             })
             .collect();
-        let clean::Item { span, name, attrs, kind, visibility, def_id } = item;
-        let inner = match *kind {
+        let docs = item.attrs.collapsed_doc_value();
+        let attrs = item
+            .attrs
+            .other_attrs
+            .iter()
+            .map(rustc_ast_pretty::pprust::attribute_to_string)
+            .collect();
+        let clean::Item { span, name, attrs: _, kind: _, visibility, def_id } = item;
```

Comment:
Am I too tired to reasonably review code right now or does `attrs` get shadowed immediately?

Reply:
You're too tired ;) `attrs: _` discards the field; and actually it has to, or otherwise there would be a borrowck error since `Box<Attributes>` isn't Copy

Reply:
Aaand I'm off to bed.